### PR TITLE
Add prac 6 files

### PR DESCRIPTION
**Pull request by a-rosa**
@AdriantoHartono
@JevanChandra
@nora-zyy
@Zunidzk

### CODE REVIEW

File: prac_06/car.py
Comment (AdriantoHartono) [inline prac_06/car.py:1]: Nice code, it is already good written👌

File: prac_06/guitar.py
Comment (AdriantoHartono) [inline prac_06/guitar.py:1]: Good code 👍

File: prac_06/guitar_test.py
Comment (AdriantoHartono) [inline prac_06/guitar_test.py:1]: I think it should be put in function, and the output I think it different from the expected from the question?😅

File: prac_06/guitars.py
Comment (AdriantoHartono) [inline prac_06/guitars.py:1]: The code already good written, but I think it will be better if the code putted in function🙏

File: prac_06/languages.py
Comment (AdriantoHartono) [inline prac_06/languages.py:1]: The code has work properly, but I think it also will be good if the code putted in function🙏

File: prac_06/programming_language.py
Comment (AdriantoHartono) [inline prac_06/programming_language.py:1]: The class already good written 👌

File: prac_06/used_cars.py
Comment (AdriantoHartono) [inline prac_06/used_cars.py:1]: The output of the code has worked as expected, nice 👌

File: prac_06/car.py
Comment (JevanChandra) [inline prac_06/car.py:1]: Great code 👍👍👍

File: prac_06/guitar.py
Comment (JevanChandra) [inline prac_06/guitar.py:1]: I would like to reference this code because I am having trouble with this section

File: prac_06/languages.py
Comment (nora-zyy) [reply prac_06/languages.py:1]: The code is well written and the presentation is clear and easy to understand.

File: prac_06/programming_language.py
Comment (nora-zyy) [reply prac_06/programming_language.py:1]: There are step-by-step instructions that are clear.

File: prac_06/guitar_test.py
Comment (JevanChandra) [inline prac_06/guitar_test.py:1]: Very short and clear!

File: prac_06/guitars.py
Comment (JevanChandra) [inline prac_06/guitars.py:1]: Nice Job!

File: prac_06/languages.py
Comment (JevanChandra) [inline prac_06/languages.py:1]: Great code!

File: prac_06/programming_language.py
Comment (JevanChandra) [inline prac_06/programming_language.py:1]: Nice work!

File: prac_06/used_cars.py
Comment (JevanChandra) [inline prac_06/used_cars.py:1]: Code works!

**Pull request merged**